runtime/indent/testdir/vim.ok

```diff
@@ -956,6 +956,7 @@ enum Digits
     NINE(3 + 3 + 3)
     const value: number
     def new(this.value)
+	return this.value
```

Comment:
```suggestion
    def new(value: number)
	this.value = value
```
This is a constructor definition (`:help E1365`).

Reply:
Modified. Thank you for pointing that out.

Reply:
@JimZhouZZY https://github.com/vim/vim/pull/16293#discussion_r1897493893